### ISSUE TITLE
issue-2 add more text for product images

### ISSUE THREAD

**Issue by EmaanIsaacsWaggie** [EmaanIsaacsWaggie/Online-shop#2]
Some product images are missing descriptive alt text. Add more meaningful descriptions for accessibility.
